lms/static/scripts/postmessage_json_rpc/server/methods.js

```diff
@@ -9,8 +9,22 @@
 /**
  * Return a Hypothesis client config object for the current LTI request.
  */
-export function requestConfig() {
+export async function requestConfig() {
   const configEl = document.querySelector('.js-config');
   const clientConfigObj = JSON.parse(configEl.textContent).hypothesisClient;
-  return clientConfigObj;
+  return Promise.resolve(clientConfigObj);
```

Comment:
Functions declared `async` always return a Promise, so you don't need to wrap the result in `Promise.resolve(..)` explicitly.